src/screens/Admin/screens/InstructorPulse/index.js

```diff
@@ -10,13 +12,15 @@ export default ({instructors}) => (
     main={
       <InstructorsList
         title={unpublishedTitleText}
-        instructors={filter(instructors, ['published_lessons', 0])}
+        instructors={filter(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', '-id'))}
```

Comment:
In the future, might also be good to have a `joined` property in the future or something that lets us know how new the instructor is so we can put new (maybe joined in the last three months or something) instructors at the top regardless of their lesson states?